src/BugsnagUnity/Native/Android/NativeClient.cs

```diff
@@ -20,6 +20,10 @@ public NativeClient(IConfiguration configuration)
             Configuration = configuration;
             Delivery = new Delivery();
             Breadcrumbs = new Breadcrumbs(NativeInterface);
+            if (configuration.AutoTrackSessions)
```

Comment:
This should preferably be `ResumeSession()` to avoid the edge case where a session has already been automatically started from an activity lifecycle callback. Also it looks like `Client.cs` might already be doing this?

Reply:
Changed to Resume 👍 
The ResumeSession call in the client only happens when if the app has been open for more than 30 seconds and it gets focus again.